vhdbuilder/scripts/automate_helpers.sh

```diff
@@ -16,6 +16,19 @@ retrycmd_if_failure() {
     done
     echo Executed \"$@\" $i times;
 }
+retrycmd_if_failure_nostdout() {
```

Comment:
probably don't need this here if we're not using it in the automation scripts

Reply:
Good point - I'll pull it out from this one.